### ISSUE TITLE
commends stuck bug

### ISSUE THREAD

**Issue by 1adadnoobsag** [ValveSoftware/Dota2-Gameplay#11505]
### Description

hi my username is 丨 卂爪 几ㄖㄖ乃 . 
my commends stuck up on 1001 .
It can't be more .
what shoud i do?

### Screenshots

![Screenshot (400)](https://github.com/ValveSoftware/Dota2-Gameplay/assets/144010452/5d9dcb2b-7c08-4c64-b1b9-281526ec2d1b)

**Comment by 1adadnoobsag** [comment]
![Uploading Screenshot (401).png…]()

**Comment by johnwickg21** [comment]
Bug , they need to take action ASAP and reset all behavior scores

**Comment by 7ReaperOfSouls7** [comment]
I confirm.
I have the same problem with likes.
Likes are no longer credited to the profile.
This problem has been in the game for about 1 week now.
Developer, please fix this issue as soon as possible.
Need a fix patch.
We hope for you.

**Comment by shaileshgarg** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/11278

**Comment by ligaapl** [comment]
Yes. I have 2172 commends a month ago and commends still not counted after profile update! ID 113184848

**Comment by kutaykartoz** [comment]
Can this be fixed already??? and fix it retroactively if possible..
This issue is here for more than a month. Valve even implemented new custom profiles and new behavior system, but didn't care about fixing one of its essential ingredient.
